translation/service_test.go

```diff
@@ -24,7 +24,7 @@ func TestSendWRP(t *testing.T) {
 
 	var expected = wrp.MustEncode(wrp.Message{
 		Type:   wrp.SimpleRequestResponseMessageType,
-		Source: "local/test",
+		Source: "local",
```

Comment:
can we change this to `dns:localhost` to make it a bit more realistic example/test? You might have to update some other part of the tests